### PR TITLE
Display related resources error message in panel

### PR DESCRIPTION
**Pull request by Pytal**
| | Screenshot |
--- | ---
**Before (Top-right toast + empty panel area)** | image image
**After** | image

### CODE REVIEW

File: src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue [hunk 1]
```diff
@@ -103,22 +114,27 @@ export default {
...
 		'has-resources',
 	],
 
 	data() {
 		return {
 			appEnabled: OC?.appswebroots?.related_resources !== undefined,
-			headerTranslated: t('Related resources'),
-			descriptionTranslated: t('Anything shared with the same group of people will show up here'),
+			headerTranslated: this.t('Related resources'),
+			descriptionTranslated: this.t('Anything shared with the same group of people will show up here'),
```
Comment (raimund-schluessler) [inline src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue:125]: This breaks the l10n extraction. See above, the strings got removed.

File: src/components/NcRelatedResourcesPanel/NcResource.vue [hunk 2]
```diff
@@ -77,7 +82,7 @@ export default {
 
 	data() {
 		return {
-			labelTranslated: t('Open link to "{resourceTitle}"', { resourceTitle: this.title }),
+			labelTranslated: this.t('Open link to "{resourceTitle}"', { resourceTitle: this.title }),
```
Comment (raimund-schluessler) [inline src/components/NcRelatedResourcesPanel/NcResource.vue:85]: Please keep it as it is, the l10n extraction does not work for the mixin.

**Review by raimund-schluessler — CHANGES_REQUESTED** [review]
Please see the l10n problems.

**Review by artonge — APPROVED** [review]

**Comment by Pytal** [comment]
@nimishavijay @jancborchardt any additional design suggestions?

**Comment by nimishavijay** [comment]
Looks good to me! ccing @jancborchardt to double check the wording :)

**Review by jancborchardt — APPROVED** [review]
Looks good to me as well! :)

File: src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue [hunk 3]
```diff
@@ -103,22 +103,33 @@ export default {
...
-			return !this.loading && this.resources.length > 0
+			if (this.loading) {
+				return false
+			}
+			return this.error ?? this.resources.length > 0
+		},
+
+		description() {
+			if (this.error) {
+				return t('Error getting related resources. Please contact your system administrator if you have any questions.')
```
Comment (nickvergessen) [inline src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue:130]: From my side of things this is not correct. When the 400 error occurs with Talks 1-1 chats there is nothing I as sysadmin was able to do about it.

Comment (nickvergessen) [reply src/components/NcRelatedResourcesPanel/NcRelatedResourcesPanel.vue:130]: Further details in the comments over there: https://github.com/nextcloud/spreed/pull/9256#issuecomment-1497089458

Now that logging is added it actually makes sense, but i find it weird that the error is not using the normal showError anymore

**Pull request merged**